config/blobs.yml

```diff
@@ -210,10 +210,10 @@ logstash/logstash-filter-translate-3.4.2.zip:
   size: 649362
   object_id: e9b87119-265c-43ce-788d-5592aabffcb3
   sha: sha256:f465635431b474abaf61c6673d7c91cb5f62b09a193d45a5622342f5a5b967c6
-logstash/logstash-input-cloudwatch_logs-1.0.5.zip:
-  size: 2153640
-  object_id: e9c13261-22d0-4cfb-6aa7-43e7414f898e
-  sha: sha256:de74e619fcb2a4d46126dd0981023bf5810ca7625580a433628abfaa68a91046
+logstash/logstash-input-cloudwatch_logs-1.0.6.zip:
```

Comment:
Shouldn't we make this blob version number match the version in the package, which is 1.1.1 now I think?

Reply:
it would make more sense, we had the version the plugin started at and the version the blob started at but lets just sync for simplicity